bberg/src/verifier_builder.rs

```diff
@@ -4,21 +4,23 @@ use crate::{
 };
 
 pub trait VerifierBuilder {
-    fn create_verifier_cpp(&mut self, name: &str, witness: &[String]);
+    fn create_verifier_cpp(&mut self, name: &str, witness: &[String], inverses: &[String]);
 
     fn create_verifier_hpp(&mut self, name: &str);
 }
 
 impl VerifierBuilder for BBFiles {
-    fn create_verifier_cpp(&mut self, name: &str, witness: &[String]) {
+    fn create_verifier_cpp(&mut self, name: &str, witness: &[String], inverses: &[String]) {
         let include_str = includes_cpp(&snake_case(name));
 
         let wire_transformation = |n: &String| {
             format!(
             "commitments.{n} = transcript->template receive_from_prover<Commitment>(commitment_labels.{n});"
         )
         };
+        // TODO: make sure the inverse commitments are not included
```

Comment:
Should this TODO also be handled in this PR?

Reply:
Ah it is handled, i can remove the todo